pytorch_lightning/accelerators/cpu_accelerator.py

```diff
@@ -64,27 +64,54 @@ def train(self):
     def training_step(self, args):
         if self.trainer.amp_backend == AMPType.NATIVE:
             with torch.cuda.amp.autocast():
-                output = self.trainer.model.training_step(*args)
+                output = self.__training_step(args)
         else:
-            output = self.trainer.model.training_step(*args)
+            output = self.__training_step(args)
+
+        return output
+
+    def __training_step(self, args):
+        batch = args[0]
+        batch = self.to_device(batch)
```

Comment:
Do you need self.to_device on cpu ?

Reply:
added this just to aling with other accelerators and to call the newly added hooks on CPU accelerator too. `batch_to_device` calls `self.prepare_batch_for_transfer` and eventually will call the added hooks.